lib/config/fileconf.go

```diff
@@ -2377,6 +2377,12 @@ type WindowsDesktopService struct {
 	ShowDesktopWallpaper bool `yaml:"show_desktop_wallpaper,omitempty"`
 	// LDAP is the LDAP connection parameters.
 	LDAP LDAPConfig `yaml:"ldap"`
+	// PKIDomain optionally configures a separate Active Directory domain
+	// for PKI operations. If empty, the domain from the LDAP config is used.
+	// This can be useful for cases where PKI is configured in a root domain
+	// but Teleport is used to provide access to users and computers in a child
+	// domain.
+	PKIDomain string `yaml:"pki_domain"`
```

Comment:
Maybe it should be a subfield of the `ldap` object?

Reply:
I don't think so. The `ldap` object should be more about "how do I connect to an LDAP server" as per

```
// LDAPConfig is the LDAP connection parameters.
```

Putting the domain and SID in there was a mistake IMO.